### PR TITLE
at least get rid of log and use color and fmt

### PR DESCRIPTION
**Pull request by kevincobain2000**
#87

### CODE REVIEW

File: cmd/gobrew/main.go [hunk 1]
```diff
@@ -2,12 +2,13 @@ package main
 
 import (
 	"flag"
-	"log"
```
Comment (kevincobain2000) [inline cmd/gobrew/main.go:5]: gets rid of `log` all together. Leaving now two output formats `fmt` and `color`.
Ref #87

File: cmd/gobrew/main.go [hunk 2]
```diff
@@ -30,18 +31,17 @@ var allowedArgs = []string{
 }
 
 func init() {
-	log.SetFlags(0)
 
 	if !isArgAllowed() {
-		log.Println("[Info] Invalid usage")
-		log.Print(usage())
+		utils.Infof("[Info] Invalid usage")
+		fmt.Println(usage())
```
Comment (juev) [inline cmd/gobrew/main.go:37]: If we use utils.Infof, why we cant to use it here?

File: gobrew.go [hunk 3]
```diff
@@ -331,7 +330,8 @@ func (gb *GoBrew) CurrentVersion() string {
 // Uninstall the given version of go
 func (gb *GoBrew) Uninstall(version string) {
 	if version == "" {
-		log.Fatal("[Error] No version provided")
+		utils.Errorln("[Error] No version provided")
+		os.Exit(1)
```
Comment (juev) [inline gobrew.go:334]: We can create new function in utils and use it here.
what do you think?

File: cmd/gobrew/main.go (hunk 2 above)
Comment (kevincobain2000) [reply cmd/gobrew/main.go:37]: Let me check

Comment (kevincobain2000) [reply cmd/gobrew/main.go:37]: Not using Infof, we don't want to add extra `==>` as a prefix to the output. It is just an indication that it is raw output. It is better to keep it this way.

Comment (juev) [reply cmd/gobrew/main.go:37]: We can use Infoln
https://github.com/kevincobain2000/gobrew/blob/master/utils/utils.go#L92

without extra symbols

File: gobrew.go (hunk 3 above)
Comment (juev) [reply gobrew.go:334]: like:
```go
func Fatalln(a ...interface{}) {
	_, _ = ColorError.Println(a...)
        os.Exit(1)
}
```

Comment (kevincobain2000) [reply gobrew.go:334]: Will change em all then

**Comment by kevincobain2000** [comment]
Fixed by #91

**Pull request closed without merging**